.eslintrc.js

```diff
@@ -1,14 +1,14 @@
 module.exports = {
   root: true,
   extends: [
-    '@react-native',
     'plugin:react/recommended',
+    'plugin:react-hooks/recommended',
     'plugin:react/jsx-runtime',
     'plugin:react-native-a11y/ios',
-    'prettier',
```

Comment:
Please remove `eslint-config-prettier` dependency too from package.json

It's usually better to not mix multiple changes in one PR, but it's up to the maintainers to comment on it. The removal of `eslint-config-prettier` can be done independently of using `react-hooks`

Reply:
Yes it is unnecessary. I am sorting through some stuff around prettier right now so I have not had a chance to update. The inclusion was because moving to plugin the rule wasn't being extended anymore. Thank you and kindly give me a bit to sort things out.